.github/workflows/build.yml

```diff
@@ -172,12 +172,21 @@ jobs:
           sudo apt update
           sudo apt install -y qemu-system-arm
 
+      - name: Install embedded-ci-client
+        run: |
+          cargo install --git https://github.com/korken89/embedded-ci.git embedded-ci-client
+
       - name: Fail on warnings
         run: sed -i 's,//deny_warnings_placeholder_for_ci,#![deny(warnings)],' src/lib.rs macros/src/lib.rs
 
       - name: Run-pass tests
-        run:
-          cargo xtask --target ${{ matrix.target }}
+        env:
+          EMBEDDED_CI_TOKEN: ${{ secrets.EMBEDDED_CI_TOKEN }}
+          EMBEDDED_CI_SERVER: ${{ secrets.EMBEDDED_CI_SERVER }}
+        run: |
+          cd examples-runner
+          cargo xtask --target ${{ matrix.target }} --runner embedded-ci
+          cargo xtask --target ${{ matrix.target }} --runner qemu
```

Comment:
I think we should split this and have two different `checkexamples` allowing them to run concurrently as this is the slowest task, we're currently not using all available 20 concurrent runners :+1: